Userland/Libraries/LibJS/Runtime/GlobalObject.cpp

```diff
@@ -202,12 +202,13 @@ JS_DEFINE_NATIVE_FUNCTION(GlobalObject::parse_float)
 {
     if (vm.argument(0).is_number())
         return vm.argument(0);
-    auto string = vm.argument(0).to_string(global_object);
+    auto input_string = vm.argument(0).to_string(global_object);
+    auto trimmed_string = input_string.trim_whitespace(TrimMode::Left);
```

Comment:
Probably doesn't matter since we get a null string in the exception case, but I still think it should go after the check below.

Reply:
yep, sorry, fixed. :)